tests/dhcp-eve-extended-option-60/test.yaml

```diff
@@ -0,0 +1,40 @@
+# *** Add configuration here ***
```

Comment:
Could you please remove this line and add
```
requires:
  min-version: 7
```
to make sure this only runs on 7 as this feature has just been added?